src/controllers/userProfileController.js

```diff
@@ -291,6 +292,7 @@ const userProfileController = function (UserProfile) {
         record.weeklySummaryNotReq = req.body.weeklySummaryNotReq
```

Comment:
Since you are using the new var, should the old ones `weeklySummaryNotReq` be deleted from this file and the whole backend repo?